exercises/isogram/example.scala

```diff
@@ -0,0 +1,19 @@
+import scala.collection.mutable
+
+object Isogram {
+  def isIsogram(s: String): Boolean = {
+    val seen = mutable.HashSet[Char]()
```

Comment:
Shall we prefer an FP solution?  
I always advice against using mutable objects and `return` if there is an alternative FP solution, so I am forced by my inner Scala-self to do it here, too. :)

And now that I have a clean conscience: I know that the example solution is not essential. You may just forget about this comment after all. :)